ch_6_GPU_Acceleration/6_2_GPU_Baichuan2-7B.md

```diff
@@ -0,0 +1,110 @@
+# 6.2 Run Baichuan 2 (7B) on Intel GPUs
+
+You can use IPEX-LLM to load any ModelScope model for acceleration on Intel GPUs. With IPEX-LLM, PyTorch models (in FP16/BF16/FP32) hosted on ModelScope can be loaded and optimized automatically on Intel GPUs with low-bit quantization (supported precisions include INT4/NF4/INT5/FP8/INT8).
+
+In this tutorial, you will learn how to run LLMs on Intel GPUs with IPEX-LLM optimizations, and based on that build a stream chatbot. A popular open-source LLM [baichuan-inc/Baichuan2-7B-Chat](https://www.modelscope.cn/models/baichuan-inc/Baichuan2-7B-Chat) is used as an example.
+
+## 6.2.1 Load Model in Low Precision
+
+One common use case is to load a ModelScope model in low precision, i.e. conduct **implicit** quantization while loading.
```

Comment:
`conduct **implicit** quantization while loading.` is a quite outdated explanation, just removed it :)